dunwoody_disaster/views/collectLootScreen.py

```diff
@@ -0,0 +1,135 @@
+from typing import Sequence
+from PySide6.QtCore import Qt
+from PySide6.QtGui import QColor
+from PySide6.QtWidgets import (
+    QWidget,
+    QVBoxLayout,
+    QLabel,
+    QCheckBox,
+    QPushButton,
+    QHBoxLayout,
+    QScrollArea,
+)
+from dunwoody_disaster.CharacterFactory import Character
+from dunwoody_disaster import Item
+import dunwoody_disaster as DD
+from dunwoody_disaster.views.meter import Meter
+
+
+class CollectLootScreen(QWidget):
+    def __init__(self, player: Character, available: Sequence[Item.Item]):
+        """
+        :param player: The player that is selecting an item.
+        :param available: A list of items the player has access to select.
+        :return: Creates a UI where a user can select which item(s)? they want to add to their inventory.
+        """
+        super().__init__()
+        self.player = player
+        self.items = available
+
+        layout = QVBoxLayout()
+        layout.setSpacing(0)
+        layout.setContentsMargins(0, 0, 0, 0)
+
+        lbl = QLabel("Loot Screen")
+        layout.addWidget(lbl)
+
+        self.capacity = Meter(QColor("red"), 0)
+        self.capacity.setStyleSheet("min-height: 50px;")
+        layout.addWidget(self.capacity)
+
+        new_items = QHBoxLayout()
+        layout.addLayout(new_items)
+
+        self.boxes: dict[QCheckBox, Item.Item] = {}
+        for item in self.items:
+            widget, box = self.create_inventory_slot(item)
+            new_items.addWidget(widget)
+            self.boxes[box] = item
+
+        scroll_area = QScrollArea()
+        old_items = QHBoxLayout()
+        item_container = QWidget()
+        item_container.setLayout(old_items)
+        scroll_area.setWidget(item_container)
+        scroll_area.setWidgetResizable(True)
+        layout.addWidget(scroll_area)
+        box = None
+        for item in self.player.get_items():
+            widget, box = self.create_inventory_slot(item)
+            old_items.addWidget(widget)
+            old_items.addSpacing(10)
+            self.boxes[box] = item
+            box.setChecked(True)
+        if box is not None:
+            self.select_item(box)
```

Comment:
I'm not super happy with how this looks. Does anyone have any refactoring ideas?